openfisca_nsw_safeguard/tests/ESS_PDRS_Estimator/WH1_F16_electric_PDRSAug24/WH1_F16_electric_PDRSAug24_installation_final_activity_eligibility.yaml

```diff
@@ -199,5 +210,129 @@
         False,
         False,
         False,
+        False,
         False
+      ]
+
+- name: Test WH1 final activity eligibility 2
+  period: 2024
+  absolute_error_margin: 0
+  input:  
+    WH1_F16_electric_PDRSAug24__equipment_replaced:
+      [ 
+        true,
+        true,
+        true,
+        true,
+        true,
+        true
+      ]
+    WH1_F16_electric_PDRSAug24__existing_equipment_removed:
+      [ 
+        true,
+        true,
+        true,
+        true,
+        true,
+        true
+      ]
+    WH1_F16_electric_PDRSAug24__equipment_installed_on_site:
+      [ 
+        true,
+        true,
+        true,
+        true,
+        true,
+        true
+      ]
+    WH1_F16_electric_PDRSAug24__qualified_install_removal:
+      [ 
+        true,
+        true,
+        true,
+        true,
+        true,
+        true
+      ]
+    WH1_F16_electric_PDRSAug24__engaged_ACP:
+      [ 
+        true,
+        true,
+        true,
+        true,
+        true,
+        true
+      ]
+    WH1_F16_electric_PDRSAug24__minimum_payment:
+      [ 
+        true,
+        true,
+        true,
+        true,
+        true,
+        true
```

Comment:
Hi @SelvaRC  sorry I think the true and false should have first letter uppercase. Please check other test files for the right format. And then can you run tests again?

Reply:
Done